File: WebSolution/PMSNGApp/src/app/activity/Models/Activity.ts
export class Activity{
     constructor(public id:number,
        public  title:string,
        public  description:string,
        public  activityType:string,
        public  projectId:number,
        public  assignedBy:number,
        public  assignedTo:number,
        public  assignDate:string,
        public  startDate:string,
        public  dueDate:string,
        public  status:string){}

}

File: WebSolution/TFLOBL/Entities/Activity.cs
namespace Transflower.TFLPortal.TFLOBL.Entities;
public class Activity
{
   public int Id{get;set;}
   public string? Title{get;set;}
   public string? ActivityType{get;set;}
   public string? Description{get;set;}
   public DateTime AssignDate{get;set;}
   public DateTime StartDate{get;set;}
   public DateTime DueDate{get;set;}
   public DateTime CreatedDate{get;set;}
   public int AssignedTo{get;set;}
   public int ProjectId{get;set;}
   public string? Status{get;set;}
   public int AssignedBy{get; set;}
  
}
